### PR TITLE
Api doc update

### PR DESCRIPTION
**Pull request by Dickow**
This pull request includes a description of all outstanding external APIs for the sim card module. 
Except for the blacklist configuration that is present on the operator profile API.

### CODE REVIEW

**Review by MrGraversen — APPROVED** [review]
🆗

File: docs/api/call-forwarding.md [hunk 1]
```diff
@@ -4,14 +4,57 @@ Using the endpoints described in this section, you are able to create, modify, a
...
 **Description**
 
 Retrieve status and details for all call forwarding rules.
 
 **Endpoint:** `GET /callforwardingrules`
 
+**Response Payload** [CallForwardingRules](/api/call-forwarding#callforwardingrules)
+
+**Example**
+```
```
Comment (grandegiraffo) [inline docs/api/call-forwarding.md]: Three back-ticks and json is  n o i c e

File: docs/api/call-forwarding.md [hunk 2]
```diff
@@ -20,6 +63,18 @@ Retrieve status and details for single call forwarding rule.
 
 **Endpoint:** `GET /callforwardingrules/{id}`
 
+**Response Payload** [CallForwardingRule](/api/call-forwarding#callforwardingrule)
+
+**Example**
+```
```
Comment (grandegiraffo) [inline docs/api/call-forwarding.md]: json

File: docs/api/call-forwarding.md [hunk 3]
```diff
@@ -44,6 +94,17 @@ targetMsisdn | String | "To" telephone number
 }
 ```
 
+**Response Payload** [CallForwardingRule](/api/call-forwarding#callforwardingrule)
+**Example**
+```
```
Comment (grandegiraffo) [inline docs/api/call-forwarding.md]: json

File: docs/api/call-forwarding.md [hunk 4]
```diff
@@ -66,10 +123,21 @@ targetMsisdn | String | The new "to" telephone number
...
```
Comment (grandegiraffo) [inline docs/api/call-forwarding.md]: json

File: docs/api/operator-profiles.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
+# Operator profiles API
```
Comment (grandegiraffo) [inline docs/api/operator-profiles.md:1]: This page is not showing on the table of contents to the left?

File: docs/api/simcard-groups.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
+# Sim card groups API
```
Comment (grandegiraffo) [inline docs/api/simcard-groups.md:1]: Page is not in the TOC?

File: docs/api/simcards-api.md [hunk 7]
```diff
@@ -81,14 +82,54 @@ Apply an action to the specified SIM card, for example "activate SIM card" or "s
...
 {
 	"simCardAction": "ACTIVATE"
 }
 ```
+```
+{
+	"simCardAction": "THROTTLE",
+	"networkSpeed": "MBIT_1"
+}
+```
```
Comment (grandegiraffo) [inline docs/api/simcards-api.md:101]: Would be nice with UNTHROTTLE example

**Review by grandegiraffo — CHANGES_REQUESTED** [review]
Looks nice. Reads very well.

File: docs/api/simcard-groups.md (hunk 6 above)
Comment (Dickow) [reply docs/api/simcard-groups.md:1]: Forgot to push those changes.

File: docs/api/simcards-api.md (hunk 7 above)
Comment (Dickow) [reply docs/api/simcards-api.md:101]: Added that.

**Review by grandegiraffo — APPROVED** [review]

**Pull request merged**